modules/richaudienceBidAdapter.js

```diff
@@ -263,3 +267,26 @@ function raiGetResolution() {
   }
   return resolution;
 }
+
+function raiGetSyncInclude(config) {
+  try {
+    let raConfig = null;
+    let raiSync = [];
+    if (config.getConfig('userSync').filterSettings != null) {
+      raConfig = config.getConfig('userSync').filterSettings
+      if (raConfig.iframe != null) {
+        if (raConfig.iframe.bidders == 'richaudience' || raConfig.iframe.bidders == '*') {
+          raiSync.push({'raiIframe': raConfig.iframe.filter})
```

Comment:
Given how `raiSync` is being used, this should be an object instead of an array. This would eliminate both the use of `filter` when getting values and the potential null pointer exceptions.

Reply:
solved, I modified the array and converted it into an object